src/status_im2/contexts/wallet/events.cljs

```diff
@@ -0,0 +1,19 @@
+(ns status-im2.contexts.wallet.events
+  (:require [utils.re-frame :as rf]
+            [taoensso.timbre :as log]))
+
+(rf/defn get-wallet-tokens
+  {:events [:wallet-2/get-wallet-tokens]}
+  [_ accounts]
+  {:json-rpc/call [{:method     "wallet_getWalletToken"
```

Comment:
just wondering why there is a difference, event with `s` and method without

Reply:
I removed the `s` in the method and event name